apps/storybook-react/.storybook/vitest.setup.ts

```diff
@@ -0,0 +1,9 @@
+import { beforeAll } from 'vitest';
```

Comment:
Setting up vitest according to storybook v9 docs https://storybook.js.org/docs/writing-tests/integrations/vitest-addon